chains.yaml

```diff
@@ -545,10 +545,23 @@
   github-organization: onomyprotocol
   github-repo: onomy
   dockerfile: cosmos
+  # TODO: Heighliner will not produce working builds until https://github.com/onomyprotocol/onomy/pull/129 merged.
+  # If PR not merged for your git-ref, then update and push to our fork, https://github.com/strangelove-ventures/onomy, with Onomy upstream.
+  # And manually run heighliner command with `-o strangelove-ventures --git-ref main --tag <semver>`.
   build-target: make install
   binaries:
     - /go/bin/onomyd
 
+# Onomy gbt: Not a chain, companion process required by validator
+- name: onomy-gbt
+  github-organization: onomyprotocol
+  github-repo: arc
+  dockerfile: rust
```

Comment:
This will work but `rust` is now deprecated. `cargo` replaces it.

Reply:
Thanks, I missed that deprecation. I'll make a quick update.